### PR TITLE
[fix bug 1278801] Redesign Firefox simple-footer to align with footer on /products

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Takes footer from `/firefox/products` page and turns it into a resuable include, replacing the "simple" footer seen on various Firefox pages such as /new.

## Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1278801

## Testing
URLs for testing:
- `/firefox/new` (both old and new templates)
- `/firefox/mobile-download/`
- `/firefox/products/`
- `/thunderbird/`

Test old & new browsers, test locales, test RTL.

## Checklist
- [x] Requires l10n changes.
- [ ] Related functional & integration tests passing.

### CODE REVIEW

File: bedrock/settings/static_media.py [hunk 1]
```diff
@@ -375,7 +376,6 @@
     },
     'firefox_onboarding_user_actions': {
         'source_filenames': (
-            'css/base/simple_footer-resp.less',
```
Comment (alexgibson) [inline bedrock/settings/static_media.py]: Note: this less file doesn't seem to be used in either on-boarding bundles, so I'm removing it. Could do with some double-checking to make sure I'm not missing something obvious.

File: bedrock/base/templates/includes/lang_switcher.html [hunk 2]
```diff
@@ -4,7 +4,14 @@
 
 {% if translations|length > 1 %}
 <form id="lang_form" method="get" action="#">
-  <label for="page-language-select">{{ _('Other languages:') }}</label>
+  <label for="page-language-select">
+    {# Remove 'en-' conditional once all locales have translated string below. #}
+    {% if LANG.startswith('en-') %}
+      {{ _('Page language:') }}
```
Comment (flodolo) [inline bedrock/base/templates/includes/lang_switcher.html]: Just to confirm, this string can be considered final, correct?

Comment (alexgibson) [inline bedrock/base/templates/includes/lang_switcher.html]: Correct, thanks.

File: media/css/firefox/new/scene2.less [hunk 3]
```diff
@@ -39,8 +39,8 @@
 
 .js .help {
     display: block;
-    padding: 0 190px;
-    margin-bottom: @baseLine * 2;
+    max-width: 40em;
```
Comment (alexgibson) [inline media/css/firefox/new/scene2.less]: This is fixing a responsive layout bug I noticed at small screen sizes on scene2 of the old `/firefox/new` template.

**Comment by alexgibson** [comment]
Layout has been approved in the bug, removing do-not-merge. Good for review

File: bedrock/settings/static_media.py (hunk 1 above)
Comment (jpetto) [inline bedrock/settings/static_media.py]: Nope, don't see it used anywhere. Probably was left in after a design change.

**Comment by jpetto** [comment]
Tested in A-grade browsers, IE 8 & 9, and RTL and all looks good. Managed to get a line break in `xh` locale, but result is pretty acceptable to me.

screen shot 2016-07-21 at 5 09 33 pm

I'll give this an r+, but will hold off on merging in case you *really* want `xh` to fit on one line. If not, I'll click the green button in the morning. 😉

**Comment by alexgibson** [comment]
Thanks @jpetto - rather than try and squish spacing any further, I went and added an extra 20px bottom margin to the logo in order to make things look a bit nicer, should the links wrap at medium viewport sizes 👍

**Comment by jpetto** [comment]
💥

**Pull request merged**